README.md

```diff
@@ -44,13 +44,17 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
 We love the NYC 311 service and the open data products that come with it. In this challenge, you will use **[311 Service Requests from 2010 to Present](https://data.cityofnewyork.us/Social-Services/311-Service-Requests-from-2010-to-Present/erm2-nwe9)** on NYC Open Data, write an ETL pipeline, and produce some data insight.
 
-This challenge has two halves, a python/bash half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python, and be callable by an executable script that takes specified arguments as inputs.
+This challenge has two halves, a python half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python.
 
 ### Python Task 1: Data Download
 
-To download 311 service request records write a proccess that 2 parameters passed from the command line and pulls the data from the API. The process shoul start with a bash script that parses the parameters of number of days and responding agency acronym then calls python code to hit the API. For example, if a user wanted to get all service request records created in the last week where DSNY is the responding agency, they would pass `7` and `DSNY` as the parameters. For this exercise, we ask that you download all 311 service requests filed the **last seven days** where **HPD** is the responding agency.  Save the data as a csv named `raw.csv` in a folder called `data`. 
+First you need to download 311 service request records. Write a python script to pulls data from the NYC Open DataAPI based two filters. The first filter is on responding agency. The second filter is an integer date filter to only get calls `n` days before the current date. The script should download the associated service requests to python memory and cache to a .csv.
```

Comment:
For this exercise, you will be working with the NYC 311 service request data. To do this, you will first need to write a python script to pull data from the NYC OpenDataAPI (**Do we want to mention socrata**) with two filters: a filter for the responding agency and a integer filter corresponding to the number of days before the current date (to get only calls within a certain day threshold i.e. the last 7 days). The script should download the associated service requests to python memory and cache to a .csv.

Reply:
I took some of your language on how to describe the filters